testsuite/tests/ui/mail_and_messages/test_message_counter.py

```diff
@@ -0,0 +1,63 @@
+"""Test of message counters in dashboard main-section tabs"""
+
+import pytest
+
+from testsuite.ui.views.admin.audience.messages import MessagesView
+from testsuite.ui.views.admin.foundation import DashboardView
+from testsuite.ui.views.devel.messages import ComposeView
+
+
+@pytest.fixture()
+def setup(navigator):
+    """Deletes all messages from inbox"""
+    admin_messages_view = navigator.navigate(MessagesView)
+    admin_messages_view.delete_all()
+
+
+def _assert_dashboard_counters(navigator, message_count, unread_count):
```

Comment:
Pls remove all underscores at beginning of variables and functions

Reply:
done